src/sections/Meshery/Meshery-integrations/Individual-Integration/IntegrationPage.style.js

```diff
@@ -0,0 +1,144 @@
+import styled from "styled-components";
```

Comment:
File names need not have to be in `CamelCase` they can be smallcase

Reply:
Do you mean it should be something like integrationpage.style.js, or are you talking about folder name, i.e. Individual-Integration?

Reply:
Yes, `integrationpage.style.js`

Reply:
But most of the other style modules follow the camel case only. Look into different folders.

Reply:
Well actually, they follow `pascalCase`

Reply:
So, `integrationPage.style.js` should be the one

Reply:
Okay!